src/app/components/Footer/index.tsx

```diff
@@ -16,71 +16,145 @@ export const supportCenterLink = 'https://help.openstax.org/s/';
 const newsletterLink = 'http://www2.openstax.org/l/218812/2016-10-04/lvk';
 
 // tslint:disable-next-line:variable-name
-const Mission = htmlMessage('i18n:footer:copyright:mission-text', Styled.Mission);
+const Mission = htmlMessage(
+  'i18n:footer:copyright:mission-text',
+  Styled.Mission
+);
 // tslint:disable-next-line:variable-name
-const Copyrights = htmlMessage('i18n:footer:copyright:bottom-text', Styled.Copyrights);
+const Copyrights = htmlMessage(
+  'i18n:footer:copyright:bottom-text',
+  Styled.Copyrights
+);
 
 // tslint:disable-next-line:variable-name
-const BareMessage: React.FunctionComponent<{id: string}> = ({id}) => <FormattedMessage id={id}>
-  {(msg) => msg}
-</FormattedMessage>;
+const BareMessage: React.FunctionComponent<{ id: string }> = ({ id }) => (
+  <FormattedMessage id={id}>{msg => msg}</FormattedMessage>
+);
 
 // tslint:disable-next-line:variable-name
-const ColumnHeadingMessage: React.FunctionComponent<{id: string}> = ({id}) => <Styled.ColumnHeading>
-  <BareMessage id={id} />
-</Styled.ColumnHeading>;
+const ColumnHeadingMessage: React.FunctionComponent<{ id: string }> = ({
+  id,
+}) => (
+  <Styled.ColumnHeading>
+    <BareMessage id={id} />
+  </Styled.ColumnHeading>
+);
 
 // tslint:disable-next-line:variable-name
-const FooterLinkMessage: React.FunctionComponent<{id: string, href: string, target?: string, rel?: string }> =
-  ({id, href, target, rel }) => <Styled.FooterLink href={href} target={target ? target : '_self'} rel={rel ? rel : ''}>
-  <BareMessage id={id} />
-</Styled.FooterLink>;
+const FooterLinkMessage: React.FunctionComponent<{
+  id: string;
+  href: string;
+  target?: string;
+  rel?: string;
+}> = ({ id, href, target, rel }) => (
+  <Styled.FooterLink
+    href={href}
+    target={target ? target : '_self'}
+    rel={rel ? rel : ''}
+  >
+    <BareMessage id={id} />
+  </Styled.FooterLink>
+);
 
 // tslint:disable-next-line:variable-name
-const SocialIconMessage: React.FunctionComponent<{id: string, href: string, Icon: React.ComponentType}> =
-  ({id, href, Icon}) => <Styled.SocialIcon aria-label={useIntl().formatMessage({id})} href={href}
-    target='_blank' rel='noopener'>
+const SocialIconMessage: React.FunctionComponent<{
+  id: string;
+  href: string;
+  Icon: React.ComponentType;
+}> = ({ id, href, Icon }) => (
```

Comment:
I know they are doing slightly different things, but the newline rules for this and https://github.com/openstax/rex-web/commit/1f1c8efa0557e37c46c7e08e47d19bd6b924bde9?w=0#diff-43286799a0a570580be50717cd6ba2a7a59d87e249a707fad745e4222daeaa26R167 seem inconsistent